IPFS_ALL_HANDS_FORMAT.md

```diff
@@ -0,0 +1,66 @@
+### IPFS All Hands Format
+
+![](https://media.giphy.com/media/JFawGLFMCJNDi/giphy.gif)
+
+## The Dream
+
+For the past couple of months, we have been discussing revamping the IPFS ALL HANDS meetings. If you want more context, please see issue https://github.com/ipfs/pm/issues/636.
+
+We want the IPFS hands-on meeting act as a space where we can showcase all of our hard work and game changing projects.The IPFS meetings give viewers a taste of some of the problems we are tackling.
+
+
+## What's in it for you? 😺
+
+The main purpose of the IPFS ALL HANDS ON CALL is to highlight the most exciting work in the broader IPFS community and disseminate high-value knowledge to the many teams improving IPFS, building on top of it, or doing relevant research. You can use this platform to:
+
+1. Attract skilled contributors to your endeavor
+
+2. Motivate the community about adding support for a new feature
+
+3. Find new partners, capabilities, or challenges for your project to work on/with.
+
+
+The IPFS ALL HANDS ON CALL also gives you a chance to practice public speaking skills. Think of it as a [Toast Masters](https://www.toastmasters.org/) for the IPFS crowd. Below are some public speaking resources.
+
+* [How to be a better speaker at tech conferences – Nick Heiner – Medium](https://medium.com/@nickheiner/how-to-be-a-better-speaker-at-tech-conferences-c441abc8d28b)
+* [11 Top Tips for a Successful Technical Presentation - Scott Hanselman](https://www.hanselman.com/blog/11TopTipsForASuccessfulTechnicalPresentation.aspx)
+* [10 Tips for Improving Your Public Speaking Skills \| Harvard Professional Development | Harvard DCE](https://www.extension.harvard.edu/professional-development/blog/10-tips-improving-your-public-speaking-skills)
+
+
+
+
+
+
+## The Format
+
+Every month we will have:
+
+* One 15 minute lightening talk (1x per month)
+* Three 5 minute talks (3x per month)
+
+In total we will have a talk every Monday. The times for the talks are:
+
+| Time (PST - UTC -CET ) |
+---|
+9:00 **17:00** 18:00 |
+
+
+If you have a talk you want to give, [sign up here](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898) **or** please email me at portia @ protocol labs dot com.
+
+ ![](https://res.cloudinary.com/blockchain-side-hustle/image/upload/v1541176158/ipfs_crew_xzbhxr.png)
+
+
+
+ ### 🥁November Scheduled Talks🥁
+
+
+
+
+| November 5th            | November 12th           | November 19th          | November 26th          |
+| ------------------------|------------------------|------------------------|-------------------------|
+| "Intro to Textile App" Sander Pick | TBD         |     TBD                |    TBD                  |
+|
+
+
+### Looking forward to all of your talk proposals!
```

Comment:
It would be great to keep a list of all the previous talks/demos (there has been many) to excite folks to do their own